### PR TITLE
SageMaker NVIDIA Triton benchmarking

### PR DESCRIPTION
**Pull request by vikramelango**
*Issue #, if available:*

*Description of changes:*  This pull request is to merge SageMaker triton benchmarking code to sagemaker-triton directory. Includes PyTorch and TensorRT benchmarking code using Amazon SageMaker Inference Recommender.

*Testing done:* Yes

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x ] I have tested my notebook(s) and ensured it runs end-to-end
- [ ] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jkroll-aws** [comment]
`sagemaker-triton/benchmark/tensor-rt/workspace/conversion_bs16_dy.txt` looks like a 1.4 MB log file - can it be removed from the PR?

**Comment by jkroll-aws** [comment]
Please run `black-nb -l 100 {notebook}.ipynb` on each notebook so that the formatting check passes.

File: sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb [hunk 1]
```diff
@@ -0,0 +1,786 @@
+{
```
Comment (aaronmarkham) [inline sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb:1]: <p>How much does it cost to run this? Should add a note about it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by jkroll-aws — DISMISSED** [review]

File: sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb (hunk 1 above)
Comment (aaronmarkham) [reply sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb:1]: <p>"cost of instances thats provisioned" --&gt; cost of instances that are provisioned</p><p>I think I was more concerned about that... if the default here is to launch a 9 instances and run inference tests... what's that cost?</p>

**Review by atqy — DISMISSED** [review]

**Review by aaronmarkham — APPROVED** [review]
LGTM

**Review by atqy — APPROVED** [review]

**Pull request merged**